data/Drake_2011/metadata.yml

```diff
@@ -0,0 +1,171 @@
+    m). Tree density was similar at both sites, 128 stems ha-1 at Site 1 and 152 stems
+    ha-1 at Site 2. The study began in January 2004 and was concluded in January 2005.
+  original_file: none, data extracted from paper.
+  notes: data extracted from paper, not raw data.
+locations:
+  Yalgorup National Park:
+    latitude (deg): -32.9
+    longitude (deg): 115.69
+    description: summer dry habitat with varying depths to water table
+    precipitation, MAP (mm): 824
+    soil type: generally calcareous, sandy and with limestone features
+contexts:
+- context_property: sampling season
+  category: temporal
+  var_in: season
+  values:
+  - value: dry
+    description: Measurements taken in the dry season, February.
+  - value: wet
+    description: Measurements taken in the wet season, October.
+- context_property: water table depth
+  category: plot
+  var_in: water_table
+  values:
+  - value: deep
+    description: The water table where plants were growing ranged from 8.15 to 7.88m
+      below the natural surface.
+  - value: shallow
+    description: The water table where plants were growing ranged from 3.01 to 1.71m
+      below the natural surface.
+traits:
+- var_in: predawn_leaf_water_potentials
+  unit_in: MPa
+  trait_name: water_potential_predawn
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: 4
+  methods: The leaf water potential (MPa) was determined before dawn (04:30-06:00
+    local standard time) during February and October in leaves from each individual
+    (n = 4 plants per size class and site) using a Scholander-type pressure chamber
+    (Soilmoisture Equipment Corp, Model 3005, Santa Barbara, CA, USA) (Scholander
+    et al. 1964). Leaves from the upper canopy of each size class were collected and
+    stored cool in snap-lock bags prior to measurement.
+- var_in: soil_to_leaf_water_potential_gradient
+  unit_in: MPa
+  trait_name: .na
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: 4
+  methods: The leaf water potential (MPa) was determined before dawn (04:30-06:00
+    local standard time) during February and October in leaves from each individual
+    (n = 4 plants per size class and site) using a Scholander-type pressure chamber
+    (Soilmoisture Equipment Corp, Model 3005, Santa Barbara, CA, USA) (Scholander
+    et al. 1964). Leaves from the upper canopy of each size class were collected and
+    stored cool in snap-lock bags prior to measurement. In the wet season, midday
+    leaf water potential (12:00-13:00 local standard time) was measured in the same
+    individuals and the soil-to-leaf water potential gradient was calculated as the
+    difference between pre-dawn and midday measurements.
+- var_in: hydraulic_conductivity_per_unit_stem_area
+  unit_in: kg/m/MPa/s
+  trait_name: stem_specific_hydraulic_conductivity
+  entity_type: population
+  value_type: raw
+  basis_of_value: measurement
+  replicates: 1
```

Comment:
I think this should be `1`, but I'm not sure. There were 9 months x 4 samples/month, and these 36 samples were used to generate a single curve, from which the value is extracted.